pkgs/development/python-modules/protobuf.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, python, buildPythonPackage
+, protobuf, google_apputils
+, disabled, doCheck ? true }:
```

Comment:
Maybe
```
{ stdenv, python, buildPythonPackage, google_apputils }:

{ protobuf, ... } @attrs:

buildPythonPackage rec {  } // attrs

---

buildProtobufPackage = callPackage .. {}
```
I think that is more flexible since then one can override `buildProtobufPackage` as if they were overriding `buildPythonPackage`.